tools/rstlint.py

```diff
@@ -126,18 +214,21 @@ def check_leaked_markup(fn, lines):
     """
     for lno, line in enumerate(lines):
         if leaked_markup_re.search(line):
-            yield lno+1, 'possibly leaked markup: %r' % line
+            yield lno + 1, 'possibly leaked markup: %r' % line
 
 
 def main(argv):
-    usage = '''\
+    usage = (
```

Comment:
What is the purpose of this change?

Reply:
I presume to have a closing ')' that lines up with 'usage' and 'try', as in some C styles with '}'.  But unneeded (...) around Python expressions is not typical style.

Reply:
Going to change this back to the original which I agree is cleaner.

Reply:
Multiline strings are one of the two places in the Python grammar where significant indentation is broken. Black tries to avoid unnecessary parentheses but is easily fooled with multiline strings that aren't aligned with the current indentation level. This is a technical limitation more than conscious design.